jsdocConfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "tags": {
+    "allowUnknownTags": true
+  },
+  "opts": {
+    "destination": "./docs"
```

Comment:
I don't know that it matters if PR #12 is merged, first. The master-dist branch will be created either way. If docs is under dist/docs, they will automatically be added to the commit and merged to master-dist.